### PR TITLE
Support Comment Node for typescript

### PR DESCRIPTION
**Pull request by alpaca-tc**
Sometimes when using texlint, we may want to ignore some lines.
In such cases, we can use [textlint-filter-rule-comments](https://github.com/textlint/textlint-filter-rule-comments) to embed keywords in comments.

```
// textlint-disable
const message = "invalid word"
// textlint-enable
```

In this PR, comments node starting with `//` are now supported.
We also added several test files to run tests.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -9,5 +9,6 @@
     "sourceMap": true,
     "strict": true,
     "target": "ES5"
-  }
+  },
+  "include": ["src/**/*"]
```
Comment (hata6502) [inline tsconfig.json:13]: Thank you :+1:

**Comment by hata6502** [comment]
Thank you for your pull request!
If the pull request is merged, it looks like be able to use textlint-filter-rule-comments.

***

Please use mocha like snapshot-testing to test the entire AST.
https://efcl.info/2018/02/02/snapshot-test/

And, please add tests the following cases. 

・Comment syntax of `/* ... */`
```ts
/* comment */
```

・Comment in JSX
```ts
<>
  {/* comment */}
  str
</>
```

・Comments in nested AST
```ts
{
  // comment-nest-1
  nest: {
    // comment-nest-2
    value: "str"
  }
}
```

**Comment by alpaca-tc** [comment]
Thanks for reviewing. I added snapshots test 😄

I haven't implemented comment extractor for `.jsx` yet because it is hard for me to investigate ts parser implementation.
I'll try to implement it on next PR.

**Comment by hata6502** [comment]
LGTM!
I'll publish new version npm package soon.

**Comment by hata6502** [comment]
textlint-plugin-jsx v1.2.0 is released! 🎉
https://www.npmjs.com/package/textlint-plugin-jsx

**Pull request merged**